src/ImageSharp/Formats/Webp/Lossless/Vp8LHistogramSet.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+#nullable disable
```

Comment:
Given that `List<Vp8LHistogram> items` is in fact a `List<Vp8LHistogram?> items`, it would be nice to enable this.

Reply:
Yeah, I only disabled it and make the `items` generic type nullable as I didn't want to introduce additional checks in callers.